pkg/controller/ground.go

```diff
@@ -1187,3 +1205,79 @@ func (op *GroundControl) podUpdate(cur, old interface{}) {
 		}
 	}
 }
+
+func (op *GroundControl) ensureStructuredAuthConfigMap(kluster *v1.Kluster) error {
+	var configContent string
+	if kluster.Spec.AuthenticationConfiguration != "" {
+		configContent = string(kluster.Spec.AuthenticationConfiguration)
+	} else {
+		authConfig := &apiserverv1beta1.AuthenticationConfiguration{}
+		gvks, _, err := apiserverScheme.ObjectKinds(authConfig)
+		if err != nil {
+			return fmt.Errorf("cannot determine GVK: %w", err)
+		}
+		authConfig.GetObjectKind().SetGroupVersionKind(gvks[0])
```

Comment:
`apiserverv1beta1.SchemeGroupVersion.WithKind("AuthenticationConfiguration")` would be more concise.

Reply:
thanks, updated